UnityProject/Assets/Scripts/Messages/Server/HealthMessages/HealthPressureMessage.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections;
+using UnityEngine;
+using UnityEngine.Networking;
+
+/// <summary>
+///     Tells client to update pressure
+/// </summary>
+public class HealthPressureMessage : ServerMessage
+{
+	public static short MessageType = (short)MessageTypes.HealthPressureStats;
+	public float pressure;
+
+	public override IEnumerator Process()
+	{
+		yield return null;
+		PlayerManager.LocalPlayerScript.GetComponent<LivingHealthBehaviour>().UpdateClientPressureStats(pressure);
```

Comment:
```suggestion
		PlayerManager.LocalPlayerScript.playerHealth?.UpdateClientPressureStats(pressure);
```
same for all three messages